src/functions/getJournal/framework/handler.ts

```diff
@@ -0,0 +1,13 @@
+import { APIGatewayProxyEvent, Context } from 'aws-lambda';
+import context, { ServiceIdentifiers } from '../application/context';
+import { JournalRetriever } from '../application/service/JournalRetriever';
+import createResponse from '../../../common/application/utils/createResponse';
+import { ExaminerWorkSchedule } from '../../../common/domain/Journal.d';
+
+const journalRetriever: JournalRetriever =
+  context.get<JournalRetriever>(ServiceIdentifiers.JournalRetriever);
+
+export async function handler(event: APIGatewayProxyEvent, context: Context) {
+  const journal: ExaminerWorkSchedule = journalRetriever.getJournal();
```

Comment:
Can we introduce some kind of logging abstraction here - in a way that we could swap as we decide on concrete implementation? 
We could use something basic (like console.{log,info,warn,err}() at this point (it supposed to end up in CloudWatch anyways), while we will decide for any more complex logging solution - unless we have a specific production ready logging lib in mind that we could use - there a lot of them to choose from on npm probably ;)

Reply:
Done, we can now do the following to inject a simple console based logger that we can easily switch out at a later point:

```typescript
  constructor(
    @inject('Logger') private loggerDep: Logger,
  ) {
    this.logger = loggerDep;
  }
```

Getting some error that's preventing me from using `ServiceIdentifiers.Logger` for this (hence the string literal).